### PR TITLE
throw usageError while orderSequentially and getPendingLocalState

### PR DESCRIPTION
**Pull request by dannimad**
## Description

>[AB297](https://dev.azure.com/fluidframework/internal/_workitems/edit/297)

## Reviewer Guidance

This PR is to prevent getting the pending local state while using orderSequentially. It is a sub task of batch validation during offline mode https://github.com/microsoft/FluidFramework/pull/11711

### CODE REVIEW

File: packages/runtime/container-runtime/src/containerRuntime.ts [hunk 1]
```diff
@@ -2832,6 +2832,10 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
             throw new UsageError("can't get state when offline load disabled");
         }
 
+        if (this._orderSequentiallyCalls !== 0) {
```
Comment (vladsud) [inline packages/runtime/container-runtime/src/containerRuntime.ts:2835]: FYI - you will have conflict with my changes in main. Not sure when it shows up in next though.
That said, your change is good, and we should merge both of our changes (my this.flush() call and your change).

**Review by vladsud — APPROVED** [review]

File: packages/test/test-end-to-end-tests/src/test/batchBreakRegression.spec.ts [hunk 2]
```diff
@@ -147,7 +147,7 @@ describeNoCompat("Batching failures", (getTestObjectProvider) => {
         await runAndValidateBatch(provider, provider.documentServiceFactory, this.timeout());
     });
     describe("client sends invalid batches ", () => {
-        itExpects("Batch end without start",
+        itExpects.skip("Batch end without start",
```
Comment (markfields) [inline packages/test/test-end-to-end-tests/src/test/batchBreakRegression.spec.ts:150]: @dannimad - Can you please ensure there's a Flaky Test work item opened in ADO for these two?  Also, next time I would recommend putting these .skips in a separate PR for clarity/accountability on having a follow-up plan. Thanks!

Comment (dannimad) [reply packages/test/test-end-to-end-tests/src/test/batchBreakRegression.spec.ts:150]: There is one for batch end without start already https://dev.azure.com/fluidframework/internal/_workitems/edit/1854/ Will create another one for force nack

Comment (markfields) [reply packages/test/test-end-to-end-tests/src/test/batchBreakRegression.spec.ts:150]: Looks like Tony's trying to address some of the flakiness now:  https://github.com/microsoft/FluidFramework/pull/11913/files

**Pull request merged**